cibuild.sh

```diff
@@ -14,6 +14,8 @@ usage()
 }
 
 restoreBuildTools(){
+    # Work around issue where NuGet can't restore  because it can't get a lock
+
```

Comment:
Whooops I typed this comment, added code, and then moved the code without the comment :sad: